docs/en/api/elements/built-in/list.mdx

```diff
@@ -576,6 +576,14 @@ bindscrolltolower?: EventHandler<ListScrollEvent>;
 
 Callback triggered when scrolling to the bottom of `<list>`. The trigger position of this callback can be controlled by [`lower-threshold-item-count`](#lower-threshold-item-count).
 
+### `update-animation`
+
+```tsx
+update-animation?: "default" | "none"
+```
+
+This property controls whether the list runs animations while the data source is being updated. It currently supports insert, delete, and update animations.
```

Comment:
add compatibility table for this attribute.  like: https://lynxjs.org/api/elements/built-in/list.html#compatibility